lib/AdapterLdap.php

```diff
@@ -264,5 +264,19 @@ public function getUsersGroupsOnFacility($spEntityId, $userId)
 		return $resultGroups;
 	}
 
+	public function getMemberStatusByUserAndVo($userId, $voId)
```

Comment:
Bad declaration of method getMemberStatusByUserAndVo($userId, $voId). You may use getMemberStatusByUserAndVo($user, $vo) and then use attributes of objects $vo and $user.